packages/confirm-dialog/theme/lumo/vaadin-confirm-dialog-styles.js

```diff
@@ -8,7 +8,7 @@ registerStyles(
     [part='header'] ::slotted(h3) {
       margin-top: 0 !important;
       margin-bottom: 0 !important;
-      margin-inline-start: calc(var(--lumo-space-l) - var(--lumo-space-m));
+      margin-inline-start: calc(var(--lumo-space-l) - var(--lumo-space-m)) !important;
```

Comment:
Sounds like we might want to add component-specific custom properties for these in the future.

Reply:
The last `!important` on `margin-inline-start` is avoidable, isn't it, by resetting only `margin-top` and `margin-bottom` in Lumo, instead of `margin`?